api/v1alpha2/cluster_types.go

```diff
@@ -26,7 +26,8 @@ const (
 	ClusterFinalizer = "cluster.cluster.x-k8s.io"
 )
 
-/// [ClusterSpec]
+// ANCHOR: ClusterSpec
```

Comment:
Neat. Is this a placeholder for an existing tool which `kubebuilder` et al. will use or one to be created?

Reply:
This is specific to `mdbook` actually, it'll work with `include` directives, example here https://github.com/kubernetes-sigs/cluster-api/pull/1365/files#diff-c7ff5bd9ca0f6c5d7884c95b78ed03f3R15